tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -366,13 +430,98 @@ def test_negative_add_image_rhev_with_invalid_name(rhev):
                 # too long string (>255 chars)
                 'name': f'img {gen_string(str_type="alphanumeric", length=256)}',
                 'uuid': rhev.image_uuid,
-                'operatingsystem': rhev.os['title'],
+                'operatingsystem': os['title'],
                 'architecture': rhev.image_arch,
                 'username': "root",
             }
         )
 
 
+@pytest.mark.on_premises_provisioning
+@pytest.mark.vlan_networking
+@pytest.mark.tier3
+@pytest.mark.skip_if_not_set('vlan_networking')
+def test_positive_provision_rhev_with_host_group(rhev, provisioning, tear_down):
+    """Provision a host on RHEV compute resource with
+    the help of hostgroup.
+
+    :Requirement: Computeresource RHV
+
+    :CaseComponent: ComputeResources-RHEV
+
+    :Assignee: lhellebr
+
+    :id: 97908521-3f4d-4207-93a3-23588b5a0a53
+
+    :setup: Hostgroup and provisioning setup like domain, subnet etc.
+
+    :steps:
+
+        1. Create a RHEV compute resource.
+        2. Create a host on RHEV compute resource using the Hostgroup
+        3. Use compute-attributes parameter to specify key-value parameters
+           regarding the virtual machine.
+        4. Provision the host.
+
+    :expectedresults: The host should be provisioned with host group
+
+    :BZ: 1777992
+
+    :customerscenario: true
+
+    :CaseAutomation: Automated
+    """
+    name = gen_string('alpha')
+    rhv_cr = ComputeResource.create(
+        {
+            'name': name,
+            'provider': 'Ovirt',
+            'user': rhev.username,
+            'password': rhev.password,
+            'datacenter': rhev.datacenter,
+            'url': rhev.url,
+            'ovirt-quota': rhev.quota,
+            'organizations': provisioning.org_name,
+            'locations': provisioning.loc_name,
+        }
+    )
+    assert rhv_cr['name'] == name
+    host_name = gen_string('alpha').lower()
+    host = make_host(
+        {
+            'name': f'{host_name}',
+            'root-password': gen_string('alpha'),
+            'organization': provisioning.org_name,
+            'location': provisioning.loc_name,
+            'pxe-loader': 'PXELinux BIOS',
+            'hostgroup': provisioning.config_env['host_group'],
+            'compute-resource-id': rhv_cr.get('id'),
+            'compute-attributes': "cluster={},"
+            "cores=1,"
+            "memory=1073741824,"
+            "start=1".format(rhev.cluster_id),
+            'ip': None,
+            'mac': None,
+            'interface': f"compute_name=nic1, compute_network={rhev.network_id}",
+            'volume': "size_gb=10," "storage_domain={}," "bootable=True".format(rhev.storage_id),
+            'provision-method': 'build',
+        }
+    )
+    hostname = '{}.{}'.format(host_name, provisioning.config_env['domain'])
+    assert hostname == host['name']
+    host_info = Host.info({'name': hostname})
+    host_ip = host_info.get('network').get('ipv4-address')
```

Comment:
If you want these chained calls to not produce `AttributeError` of  `AttributeError: 'NoneType' object has no attribute 'get'`, you should provide a default empty dict in all `get()` calls that are followed by another `.get()`.

```
host_ip = host_info.get('network', {}).get('ipv4-address')
```

Not a required change, but it would cause the test to ERROR early here with an AttributeError, instead of failing with an AssertionError when `host_ip` is compared to `ip_addr`. The other two asserts against `does_vm_exist` and the MAC addr comparison won't run.